select2.js

```diff
@@ -797,9 +797,7 @@ the specific language governing permissions and limitations under the Apache Lic
             }
 
             installKeyUpChangeEvent(search);
-            search.on("keyup-change input paste", this.bind(function () {
-                this.updateResults(true)
-            }));
+            search.on("keyup-change input paste", this.bind(this.updateResults));
```

Comment:
Yes not nice for me too.
I just reverted to version that works. and dont want touch "their magic"